images/Makefile

```diff
@@ -91,6 +90,21 @@ endef
 
 makefile-deps := Makefile $(obj)/Makefile
 
+#
+# Generate descriptor.pb-c.c and descriptor.pb-c.h to compile opts.proto.
+PROTOBUF_DIR := images/google
+DESCRIPTOR_DIR := $(PROTOBUF_DIR)/protobuf
+$(DESCRIPTOR_DIR)/descriptor.pb-c.c: /usr/include/google/protobuf/descriptor.proto
```

Comment:
should we do something like:
GOOGLE_INCLUDE=$(shell pkg-config protobuf --variable=includedir)/google/protobuf